bundles/org.eclipse.jface/src/org/eclipse/jface/action/StatusLineManager.java

```diff
@@ -154,9 +156,13 @@ public IProgressMonitor getProgressMonitor() {
 			@Override
 			public void beginTask(String name, int totalWork) {
 				if (taskStarted != null) {
-					throw new IllegalStateException(
+					/// TODO: maybe even logging is too much as long as every SubMonitor.convert()
```

Comment:
Yes, this could be an issue too. 

If you can commit to work on reported bugs during 4.30 release time (based on reported log entries), we can keep this AS IS till M3 and disable logging or hide it behind a tracing option for the final M3 version.

If you can't, I would add a boolean flag and disable logging right now.